api/log.go

```diff
@@ -83,6 +83,7 @@ func (h *LogHandlers) logHandler(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 		if local && err == repo.ErrNotFound {
+			// TODO (b5): This should be 403 not 500
```

Comment:
Oops! I meant 404, my mistake! It's fine like this, just having a TODO is enough.